packages/react/src/primitives/Pagination/PaginationItem.tsx

```diff
@@ -84,7 +88,12 @@ export const PaginationItem: React.FC<PaginationItemProps> = ({
       return (
         <View as="li">
           <Button
-            className={ComponentClassNames.PaginationItemButton}
+            className={classNames(
+              ComponentClassNames.PaginationItemButton,
+              isDisabled
```

Comment:
Shouldn't we add `${ComponentClassNames.PaginationItemButton}--link` here as well according to the CSS changes you made